scripts/utils/skill-logger.sh

```
@@ -0,0 +1,57 @@
+
+# Setup automatic logging on script exit
+# Args: $1=skill_name, $2=action
+setup_skill_logging() {
+  local skill=$1
+  local action=$2
+
+  # Initialize logging
+  init_skill_logging
+
+  # Setup trap to log on exit
+  trap "log_skill_invocation $skill $action" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)